### PR TITLE
Update atlite and geo packages

### PR DESCRIPTION
**Pull request by timtroendle**

### CODE REVIEW

File: envs/hydro.yaml [hunk 1]
```diff
@@ -3,21 +3,16 @@ channels:
     - conda-forge
 dependencies:
     - python=3.7
```
Comment (brynpickering) [inline envs/hydro.yaml:5]: Does it make sense to remain consistent between sub-environments with the python version?

File: src/hydro/runoff.py [hunk 2]
```diff
@@ -1,72 +1,34 @@
...
 
-X_RANGE = slice(-10.75, 34.75)
-Y_RANGE = slice(71.25, 34.5)
 
-
-def runoff(path_to_cutout, year):
+def runoff(path_to_cutout, year, x_min, x_max, y_min, y_max):
     """Retrieve runoff data from ERA5 as atlite cutout."""
+    logging.basicConfig(level=logging.INFO)
     path_to_cutout = Path(path_to_cutout)
```
Comment (brynpickering) [inline src/hydro/runoff.py:11]: As with inflow_m3.py, you probably don't need this any more

File: src/hydro/runoff.py [hunk 3]
```diff
@@ -1,72 +1,34 @@
...
-
-    monkeypatch_atlite_to_download_runoff_only()
+    x_range = slice(x_min, x_max)
```
Comment (brynpickering) [inline src/hydro/runoff.py:12]: does `slice` include the last element, or not?

File: src/hydro/runoff.py [hunk 4]
```diff
@@ -1,72 +1,34 @@
 """Retrieve runoff data from ERA5 as atlite cutout."""
+import logging
 from pathlib import Path
 
 import atlite
-import xarray as xr
 
-X_RANGE = slice(-10.75, 34.75)
```
Comment (brynpickering) [inline src/hydro/runoff.py:7]: nice

**Review by brynpickering — COMMENTED** [review]
Looks like a nice cleanup (on atlite's side of things, as much as anything!). I will assume there's been no breaking changes from the version updates ;)

File: envs/hydro.yaml (hunk 1 above)
Comment (timtroendle) [reply envs/hydro.yaml:5]: Good point. Will update everything to 3.8.

File: src/hydro/runoff.py (hunk 3 above)
Comment (timtroendle) [reply src/hydro/runoff.py:12]: Not sure if that's a "slice" feature or an "atlite" feature, but it does.

**Comment by timtroendle** [comment]
Ready for you to review again @brynpickering. I've added a changelog and will use that for the other pull requests as well.

**Review by brynpickering — APPROVED** [review]

**Pull request merged**